web/src/test/java/no/nav/modiapersonoversikt/rest/persondata/AdresseMappingTest.kt

```diff
@@ -33,6 +33,7 @@ internal class AdresseMappingTest {
         val tredjepartsPerson = gittTredjepartsperson().copy(
             bostedsadresse = HentTredjepartspersondata.Bostedsadresse(
                 folkeregistermetadata = null,
+                coAdressenavn = null,
```

Comment:
Utvid gjerne denne testen ved å sette `coAdressenavn` til noe slik at vi sikrer oss for fremtiden